### PR TITLE
Add subscribe button to global and sticky navs

### PR DESCRIPTION
**Pull request by joshdarby**
## Changes

This pull request makes the following changes:

- Adds a "Subscribe" button to the global nav and sticky nav next to the "Donate" button 

(ignore the missing icon font in these screenshots)
Desktop:
Screen Shot 2020-05-04 at 1 29 58 PM

Sticky:
Screen Shot 2020-05-04 at 1 30 05 PM

Mobile:
Screen Shot 2020-05-04 at 1 31 17 PM

## Why

For #83 / Giving Tuesday

## Testing/Questions

Features that this PR affects:

- Global nav, sticky nav, mobile nav

Questions that need to be answered before merging:

- [ ] Is this PR targeting the correct branch in this repository?
- [ ] Are the colors ok?
- [ ] The donate button has an icon inside of it. Does the subscribe button need an icon as well?
- [ ] The donate button is hidden on the mobile menu. Should the subscribe button be hidden as well?

Steps to test this PR:

1. View the subscribe button on the global nav on desktop
2. View the subscribe button on the sticky nav on desktop
3. View the subscribe button on the mobile nav

### CODE REVIEW

File: wp-content/themes/borderzine/less/inc/header.less [hunk 1]
```diff
@@ -17,4 +17,41 @@
...
+    a {
+        padding: 24px 7px;
+        color: @orange;
+        &:hover {
+            text-decoration: none;
+            color: @orange;
+        }
+    }
+    &:hover {
+        background: darken(@white, 5%);
```
Comment (benlk) [inline wp-content/themes/borderzine/less/inc/header.less:36]: I'm worried that this background-color darkening is not enough to provide a legible darkening. Can it go darker?

File: wp-content/themes/borderzine/less/inc/header.less [hunk 2]
```diff
@@ -17,4 +17,41 @@
...
+    }
+}
+.subscribe {
+    .subscribe-link {
+        span {
+            padding: 6px 8px;
```
Comment (benlk) [inline wp-content/themes/borderzine/less/inc/header.less:42]: should be 4px 8px to match the donate button.

File: wp-content/themes/borderzine/less/inc/header.less (hunk 1 above)
Comment (joshdarby) [reply wp-content/themes/borderzine/less/inc/header.less:36]: Sure! Let's try 10%

File: wp-content/themes/borderzine/less/inc/header.less (hunk 2 above)
Comment (joshdarby) [reply wp-content/themes/borderzine/less/inc/header.less:42]: Whoops. Good catch.

Comment (joshdarby) [reply wp-content/themes/borderzine/less/inc/header.less]: Fixed in https://github.com/INN/umbrella-borderzine/pull/84/commits/9835e2a610507b1e052cdb24c4cd4bbaf12ceb89

File: wp-content/themes/borderzine/less/inc/header.less (hunk 1 above)
Comment (joshdarby) [reply wp-content/themes/borderzine/less/inc/header.less]: Updated to 10% in https://github.com/INN/umbrella-borderzine/pull/84/commits/9835e2a610507b1e052cdb24c4cd4bbaf12ceb89

**Comment by joshdarby** [comment]
> Needs hover styles for the Subscribe button in the sticky nav, but otherwise good to ship.

You're right. https://github.com/INN/umbrella-borderzine/pull/84/commits/4ec5dd5e9d63d886dbf0eee654150a1b6caf7c4c switches the `darken` function to `lighten` and bumps it up to `15%`

**Pull request merged**